contributors/devel/pull-requests.md

```diff
@@ -81,21 +79,24 @@ For PRs with a release note:
     Your release note here
     ```
 
-For PRs without a release note:
+For PRs that require additional action from users switching to the new release, include the string "action required" (case insensitive) in the release note:
+
+    ```release-note
+    action required: your release note here
```

Comment:
As above:
"[action required] Addresses issue #(link to issue, feature description, other document). Users upgrading to this version must (provide details about user action required)."